browser/extensions/brave_component_extension.h

```diff
@@ -26,6 +29,8 @@ class BraveComponentExtension : public ComponentsUI {
                 const std::string& component_base64_public_key);
   static bool Unregister(const std::string& component_id);
 
+  std::string component_id() const;
```

Comment:
no objection about this but I don't think it's used?

Reply:
Good catch, I'll remove that it turned out not to be necessary.